academy-2020-piemdb/scripts/seed-ddb.js

```diff
@@ -56,7 +56,7 @@ fs.readFile('./PieMDB.json', 'utf-8', async (error, contents) => {
   for (let i = 0; i < batchedPies.length; i += 1) {
     const pieParams = {
       RequestItems: {
-        'PieMDB-database': batchedPies[i].map((pie) => ({
+        'PieMDB-database-dev': batchedPies[i].map((pie) => ({
```

Comment:
I notice this is different to the table name in PieMDB.json and it appears to me that shouldn't be the case.
What do you think about a default table name of `PieMDB-database` and replace it for different environments with a `TABLE_NAME` environment variable (although we should probably use the name of whatever was used to create the table regardless of whether it's an env variable or not, i.e use `table.tableName`)?